PulsePlayer/PulsePlayer/ios/SkinViewController.m

```diff
@@ -314,6 +318,16 @@ - (IBAction)closeButtonPressed {
   [self dismissViewControllerAnimated:YES completion:nil];
 }
 
+- (IBAction)playerVolumeSliderChanged:(id)sender {
+    //    AVPlayer has no volume property and requires the use of the system volume setting which can be controlled only by the hardware switch or an MPVolumeView
+    //   volumecontrol reference- https://github.com/12Rockets/VolumeControl
+    //    MPVolumeView only works on an actual device and not on a simulator
+    MPVolumeView *volumeView = [[MPVolumeView alloc] init];
+    self.player.volume = _volumeSlider.value;
+    [_volumeSlider setContinuous: NO];
```

Comment:
`setContinuous` only needs to be set once AFAIK. Here it gets set to the same value every time the volume is changed, which seems unnecessary.